src/components/Cards/Login.vue

```diff
@@ -98,4 +98,8 @@ export default {
 </script>
 
 <style lang="css" scoped>
+.button-padding{
+  padding-right: 24px;
```

Comment:
I wonder if we should use the normal veutify padding helpers here (rather than tweaking the css ourselfs. 

Have a look at https://vuetifyjs.com/en/styles/spacing/#how-it-works